tests/test_pep.py

```diff
@@ -36,4 +54,4 @@
 
 ])
 def test_bump_version(rule, old, new):
-    assert bump_version(rule=rule, version=old, scheme='pep') == new
+    assert bump_version(rule=rule, version=old, scheme='pep') == new, "Rule {} failed for '{}'!".format(rule, old)
```

Comment:
up to you but pytest already shows parameters for the failed case.

Reply:
For me, this change made it easier to detect, which rule was violated because the output was shorter and clearer.